src/workflow/RunEnergyPlus.cpp

```diff
@@ -201,16 +217,12 @@ void OSWorkflow::runEnergyPlus() {
     int result = 0;
 
     if constexpr (useBoostProcess) {
-      detailedTimeBlock("Running EnergyPlus", [this, /*&cmd,*/ &result, &runDirPath, &runDirResults, &inIDF] {
+      detailedTimeBlock("Running EnergyPlus", [this, /*&cmd,*/ &result, &runDirPath, &runDirResults, &inIDF, &stdout_ofs] {
         // result = std::system(cmd.c_str());
         namespace bp = boost::process;
         bp::ipstream is;
-        // boost::iostream::file_descriptor_sink fl(runDirPath / "stdout-energyplus");
-        // boost::filesystem::ofstream ofs(runDirPath / "stdout-energyplus");
-        std::ofstream stdout_ofs(openstudio::toString(runDirPath / "stdout-energyplus"), std::ofstream::trunc);
         std::string line;
-        // bp::child c(cmd, bp::std_out > is);
-        bp::child c(runDirResults.energyPlusExe, inIDF.filename(), bp::std_out > is);
+        bp::child c(runDirResults.energyPlusExe, inIDF.filename(), (bp::std_out & bp::std_err) > is);
```

Comment:
Pipe both stdout and stderr for E+, because the final `EnergyPlus Completed Successfully.` is printed to stderr and we want to swallow it as well.